sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -379,19 +378,16 @@ static void _netif_list_ipv6(ipv6_addr_t *addr, uint8_t flags)
                 break;
         }
     }
-    line_thresh = _newline(0U, line_thresh);
+    puts("");
```

Comment:
These seems they would change the output, as the previous version looks like its printing `\n (many spaces) `.
Not sure if it is wanted to replace it with only `\n`.

Reply:
Same goes here.